src/app/data/AbsDeaths.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
+
+export default function useAbsDeaths() {
+  const [absDeathsData, setAbsDeathsData] = useState(null);
+
+  useEffect(() => {
+    const fetchAbsDeaths = async () => {
+      const res = await fetch('/api/who-data');
+      const jsonData = await res.json();
+      // console.log(jsonData.data.value[0]);
+      setAbsDeathsData(jsonData.data);
+    };
```

Comment:
This can be hoisted to outside `useEffect`.

Reply:
Done.